lab/13-npm/classes.js

```diff
@@ -0,0 +1,192 @@
+'use strict';
+
+class Person {
+    constructor(cnp){
+        this.cnp = cnp;
+    }
+};
+
+class Customer extends Person {
+    constructor (cnp, money){
+        super(cnp);
+        this.money = money;
+        this.shoppingCart = [];
+    }
+
+    addToCart(product, quantity, available_products){
+        var available = false;
+        var today = new Date();
```

Comment:
Never use `var` in your code. Use only `let` or `const`. 
In this case `const` is more appropriate since you don't plan to change its value.